s2d-oms-mgmt-solution/prereqs/azureprereqdeploy.json

```diff
@@ -0,0 +1,30 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {},
+    "variables": {
+        "workspacename": "[concat('omswrkspc',uniqueString(resourceGroup().id))]",
+        "location": "West Europe"

```

Comment:
In azuredeploy.json, values for locations are restricted, which is why we hard coded it to  'West Europe' which comes under the allowed values specified.